roles/nuage_master/tasks/certificates.yml

```diff
@@ -10,7 +10,7 @@
 
 - name: Create the req file
   command: >
-    openssl req -key "{{ nuage_ca_master_rest_server_key }}" -new -out "{{ nuage_mon_rest_server_crt_dir }}/restServer.req" -subj "/CN={{ ansible_nodename }}"
+    openssl req -key "{{ nuage_ca_master_rest_server_key }}" -new -out "{{ nuage_mon_rest_server_crt_dir }}/restServer.req" -subj "/CN={{ openshift_master_cluster_hostname }}"
```

Comment:
Since nuage_master requires openshift_facts and should not be run on a host that openshift_master has not been applied to, you can use: `openshift.master.cluster_hostname | default(openshift.common.hostname)`